eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,20 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+
+    <!-- These are cases where the component chooses, when built in isolation, to 
+         remain dependent on stable versions. These versions may not have source build intermediates associated with them.
+         They cannot be SBRPs because they actually provide functionality. In full source build, these versions are
```

Comment:
Can we expand out the SBRP initialism here? Most roslyn developers won't know what this means.